packages/aws-cdk-lib/aws-kms/lib/key.ts

```diff
@@ -434,6 +434,13 @@ export interface KeyProps {
    */
   readonly enableKeyRotation?: boolean;
 
+  /**
+   * The period between each automatic rotation.
+   *
+   * @default - 365 days.
```

Comment:
```suggestion
   * @default - set by CFN to 365 days.
```

This clarifies to future contributors that the value is intentionally not set to 365 by the CDK, and instead defers to CFN for that value.